tasks/st0317a_clean_cert.rb

```diff
@@ -62,4 +66,4 @@ def clean_cert(agent, cmd)
 
 puts results.to_json
 
-exit(results.values.all? { |v| v[:result] == 'Certificate removed' }) ? 0 : 1
+exit(results.values.all? { |v| v[:result] == '***THIS TASK IS NOW DEPRECATED - Please see README*** - Certificate removed' }) ? 0 : 1
```

Comment:
Same as the others, we shouldn't add the deprecation to the results of each certname we try to clean. Please remove it from here.